src/app/components/home/home.component.ts

```diff
@@ -42,13 +42,16 @@ export class HomeComponent implements OnInit {
 
   public windowWidth = window.innerWidth;
 
+  @ViewChild('tooltip') t
```

Comment:
If you're going to leave it in the demo app, you might name it `tooltip` instead of `t` for greater clarity